### PR TITLE
[ja]Apply changes in #29770

### PR DESCRIPTION
**Pull request by YuikoTakada**
This PR fix Japanese website.
This PR applies the change which was added by https://github.com/kubernetes/website/pull/29770/files to Japanese website.
This PR fixes below page.
https://kubernetes.io/ja/docs/concepts/storage/persistent-volumes/#%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B9%E3%83%A2%E3%83%BC%E3%83%89

English page:
https://kubernetes.io/docs/concepts/storage/_print/#access-modes

fixes: #30783

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @nasa9084

**Comment by atoato88** [comment]
It's better to write in English for Non-Japanese speaker.

This PR update updates in Japanese documentation.
This is same update on https://github.com/kubernetes/website/pull/29770/files to Japanese contents.
Target pages(Japanese contents) is following:
- https://kubernetes.io/ja/docs/concepts/storage/persistent-volumes/#%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B9%E3%83%A2%E3%83%BC%E3%83%89

Original target page(English contents) is following:
- https://kubernetes.io/docs/concepts/storage/_print/#access-modes

**Comment by atoato88** [comment]
/lgtm
Thank you to quick translation to Japanese!

File: content/ja/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -376,9 +376,19 @@ PersistentVolumeは、リソースプロバイダーがサポートする方法
...
+`ReadOnlyMany`
+: ボリュームは多数のNodeで読み取り専用としてマウントできます
+
+`ReadWriteMany`
+: ボリュームは多数のNodeで読み取り/書き込みとしてマウントできます
+
+`ReadWriteOncePod`
+: ボリュームは、単一のPodで読み取り/書き込みとしてマウントできます。クラスタ全体で1つのPodのみがそのPVCの読み取りまたは書き込みを行えるようにする場合は、ReadWriteOncePodアクセスモードを使用します。 これは、CSIボリュームとKubernetesバージョン1.22以降でのみサポートされます。
+
+これについてはブログ [Introducing Single Pod Access Mode for PersistentVolumes](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/) に詳細が記載されています。
```
Comment (nasa9084) [inline content/ja/docs/concepts/storage/persistent-volumes.md:391]:
```suggestion
これについてはブログ[Introducing Single Pod Access Mode for PersistentVolumes](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/)に詳細が記載されています。
```

**Comment by nasa9084** [comment]
Please create issue at first.
c.f. https://kubernetes.io/ja/docs/contribute/localization/

**Comment by YuikoTakada** [comment]
@nasa9084 Thank you for your comments. I'll fix it.

File: content/ja/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -376,9 +376,19 @@ PersistentVolumeは、リソースプロバイダーがサポートする方法
...
+: ボリュームは単一のNodeで読み取り/書き込みとしてマウントできます
+
+`ReadOnlyMany`
+: ボリュームは多数のNodeで読み取り専用としてマウントできます
+
+`ReadWriteMany`
+: ボリュームは多数のNodeで読み取り/書き込みとしてマウントできます
+
+`ReadWriteOncePod`
+: ボリュームは、単一のPodで読み取り/書き込みとしてマウントできます。クラスタ全体で1つのPodのみがそのPVCの読み取りまたは書き込みを行えるようにする場合は、ReadWriteOncePodアクセスモードを使用します。 これは、CSIボリュームとKubernetesバージョン1.22以降でのみサポートされます。
```
Comment (nasa9084) [inline content/ja/docs/concepts/storage/persistent-volumes.md:389]:
```suggestion
: ボリュームは、単一のPodで読み取り/書き込みとしてマウントできます。クラスタ全体で1つのPodのみがそのPVCの読み取りまたは書き込みを行えるようにする場合は、ReadWriteOncePodアクセスモードを使用します。これは、CSIボリュームとKubernetesバージョン1.22以降でのみサポートされます。
```

Comment (YuikoTakada) [reply content/ja/docs/concepts/storage/persistent-volumes.md:389]: Thank you for your comments. I'll fix it.

**Comment by nasa9084** [comment]
/approve
Thank you for your contribution~

**Pull request merged**